templates/docs/examples/patterns/hero/hero-signpost-deprecated.html

```diff
@@ -0,0 +1,28 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Hero / Signpost logo (deprecated){% endblock %}
+{% block standalone_css %}patterns_hero{% endblock %}
+
+{% set is_paper = True %}
+{% block content %}
+
+<section class="p-section--hero">
+  <div class="row--25-75">
+    <div class="col">
+      <img src="https://assets.ubuntu.com/v1/56e5a802-spark-logo.png" width="100" />
```

Comment:
Removed this instance of it since it's the only one being touched by this PR, but these are spread throughout the codebase (currently 58 instances)- probably these should be removed as a separate code-cleanup effort

![image](https://github.com/user-attachments/assets/60a3a508-2861-4b47-8e99-7d82a34cc514)